docs/service_lifecycle.md

```diff
@@ -0,0 +1,62 @@
+# Service Lifecycle
+
+This defines the current service lifecycle stages, this isn't specific to the hosting platform
+and will no doubt move to somewhere more centric and referenced. However, this is a starting 
+point to clearly communication the phases of our evolution. 
+
+## Alpha
+
+Alpha is an experimental phase to test the hypothesis, by building prototypes to validate
+the direction and the intention of the service. It is there to explore ways of achieving and meeting
+the user needs in the right way. 
+
+There is more information on this [here](https://www.dta.gov.au/standard/service-design-and-delivery-process/) and
+[here](http://ausdto.github.io/service-handbook/alpha/1-introduction/1-1-what.html)
+
+
+## Beta
+
+This is the phase where software is more feature complete, it is aimed at being functional and meeting the definition
+of a Minimal Viable Product, (MVP). The service is accessible and secured and meets a lot of the production requirements and needs,
+however, there will be a assessment of the service through closed user groups; to evaluate that it meets the needs as expected.
+SLA's and SLO's may differ from the user expectation of a live service and the availability expectation may not be to the standards
+people would want of a live service. 
+
+## Production
+
+Once the product's have gone through the beta process, (which is to identify that the MVP is viable), the service will transition
```

Comment:
....the service will transition to production. In production, there will be more rigour around......